cli/ssh_test.go

```diff
@@ -21,6 +21,11 @@ import (
 	"testing"
 	"time"
 
+	"github.com/coder/coder/v2/provisioner/echo"
+	"github.com/google/uuid"
+
+	"github.com/coder/coder/v2/coderd/rbac"
```

Comment:
Sloppy imports, it's too bad our formatter doesn't fix this 😔

Reply:
Hmm, not sure why goimports is not fixing this. Got a new PC, tried to configure my goimports a little better, but still failing to fix everything :cry:

Reply:
You could try [gofumpt](https://github.com/mvdan/gofumpt), although that shoudl already be used by Makefile 🤔